### DISCUSSION TITLE
Ring cameras will not keep live view in Apple Homekit for more than 30 secs

### DISCUSSION THREAD

**Discussion by gburlile** [dgreif/ring#668] [Q&A] [answered]
Hello!  My first time posting here.  I have multiple Ring cameras outside my house.  And using Hoobs and this Ring plugin have successfully added these cameras so they show in the Apple Home app.  HOWEVER, anytime I click on one of the cameras to get a live view, after 30 seconds of viewing the live view the camera video is replaced with the following message "No Signal This camera is not responding".  This happens no matter the camera and no matter what Apple device I'm using to view the camera feed on the Apple Home app.  Any help would be appreciated.

**Comment by dgreif** [comment, marked as answer]
This is a known issue that I haven't tracked down a fix for yet.  See https://github.com/dgreif/ring/issues/479
